### DISCUSSION TITLE
Backup skipped -- can't re-add repository

### DISCUSSION THREAD

**Discussion by oldmanagain** [borgbase/vorta#1755] [FAQ] [answered]
I've been trying out Vorta because I'm no good on terminal stuff.  It's run perfectly for 2 days, then, this am checking the log I see it didn't run with the following errors:

vorta.scheduler - ERROR - Conditions for backup not met. Aborting.
vorta.scheduler - ERROR - Your repo passphrase was stored in a password manager which is no longer available.
Try unlinking and re-adding your repo.

So I tried the unlinking and re-adding and get this error:

vorta.borg.borg_job - ERROR - passphrase supplied in BORG_PASSPHRASE, by BORG_PASSCOMMAND or via BORG_PASSPHRASE_FD is incorrect.

I have no idea how to fix this---my gut says completely remove both Vorta and Borg then start over.

Any suggestions short of that?

OS: Linux Mint 21.1
Vorta version: 0.8.3
Installed from: Software Manager

**Comment by real-yfprojects** [comment, marked as answer]
This seems to be a problem with the password supplied by the password manager. Which one do you use and did you modify any configuration regarding it? Did you modify the keyring entry created by Vorta?

Can you confirm the password supplied to vorta is correct by running borg in the terminal?

**Comment by oldmanagain** [comment]
I suppose the password manager is the one installed with Linux.  I've made no modifications to any passwords or keyring.  How do I run borg in a terminal?

**Reply by jetchirag** [reply]
Open up your terminal and run command `vorta`

**Comment by oldmanagain** [comment]
Okie dokie...thanks to those replying it provided a path to figure out what went wrong.

My pc generally runs 24/7 with an occasional restart.  I'm set at on autologon---no password screen.  The "password manager" is the key info---I've never looked at that app (Seahorse). I found it in the mint menu->Passwords and Keys".  Poking around in it revealed that the Default keyring was locked.  I unlocked with my keyring password, launched Vorta and was able to add existing repository and perform the backup that was missed.  So, the keyring was locked when Vorta tried to backup.

I sorta looked around the web to maybe find instructions on how to unlock the keyring so I could put that in Vorta's Shell Commands --- no luck.  Right now I have in Startup apps Seahorse.  It doesn't unlock the keyring but does provide me with a reminder to unlock the keyring when I restart the system.

(this kind of stuff is a reminder that 'old me' shouldn't be messing around)

Thanks Folks
